_episodes_rmd/05-data-structures-part2.Rmd

```diff
@@ -5,13 +5,20 @@ exercises: 10
 questions:
 - "How can I manipulate a dataframe?"
```

Comment:
I think this is as good a place as any to start fixing the inconsistent use of `data.frame` and data frame throughout these lessons. We had a long discussion this a while back on #145 and I think the conclusion was to use "data frame" when referring to the thing and `data.frame` only when referring to the function `data.frame()`.  So in that vein, could you change the references to the thing throughout to "data frame" instead of as above "dataframe" or as you have below `data.frame`. One of these days I'm going to go through and do the same thing for all the other lessons as well.